src/ontology/uberon-edit.obo

```diff
@@ -224708,6 +224685,44 @@ relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0007365 ! rumen
 property_value: dcterms-date 2023-02-08T08:53:48Z xsd:dateTime
 
+[Term]
+id: UBERON:8600000
+name: lobular bronchiole
+def: "A conductive airway that branches into terminal bronchioles." [PMID:16543587, PMID:27611890]
+synonym: "preterminal bronchiole" EXACT [PMID:16543587]
+is_a: UBERON:0002186 ! bronchiole
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-08T13:31:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8600001
+name: epithelium of lobular bronchiole
+def: "An epithelium of the lobular bronchiole." [FMA:62497]
+synonym: "epithelium of preterminal bronchiole" EXACT [PMID:16543587]
+synonym: "lobular bronchiole epithelium" EXACT [FMA:62497]
+intersection_of: UBERON:0000483 ! epithelium
+intersection_of: part_of UBERON:8600000 ! lobular bronchiole
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-10T14:26:09Z xsd:dateTime
+
+[Term]
+id: UBERON:8600002
+name: mucosa of lobular bronchiole
+def: "A mucosa that is part of a lobular bronchiole." [FMA:263230]
+intersection_of: UBERON:0000344 ! mucosa
+intersection_of: part_of UBERON:8600000 ! lobular bronchiole
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-10T14:48:35Z xsd:dateTime
+
+[Term]
+id: UBERON:8600003
+name: smooth muscle tissue of lobular bronchiole
+def: "A portion of smooth muscle tissue that is part of a lobular bronchiole." [FMA:261081]
```

Comment:
Unsure what "A portion of" is meant to convey... is there another portion of smooth muscle tissue in the lobular bronchiole that is not included in this class?